sonarqube-sample-java/src/main/java/com/consoleconnect/mw/sonarqube/sample/entity/User.java

```diff
@@ -13,6 +13,8 @@
 	private String password;
 
 	private Collection<Role> roles;
+	
+	private int a;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "a" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1068)

Reply:
ACK